package/gargoyle/files/www/js/bandwidth.js

```diff
@@ -909,3 +956,64 @@ function deleteData()
 	var param = getParameterDefinition("commands", commands.join("\n"))  + "&" + getParameterDefinition("hash", document.cookie.replace(/^.*hash=/,"").replace(/[\t ;]+.*$/, ""));
 	runAjax("POST", "utility/run_commands.sh", param, stateChangeFunction);
 }
+
+function setCustBWMonVisilbility()
```

Comment:
Really minor typo in function "setCustBWMonVisilbility". Probably won't break anything, just pointing it out before it gets merged.

Reply:
Whoops, fixed.